diff --git a/qtsystems.pro b/qtsystems.pro
index b3a101d..5d3f75d 100644
--- a/qtsystems.pro
+++ b/qtsystems.pro
@@ -15,6 +15,7 @@ module_qtsystems_tests.CONFIG = no_default_install
 module_qtsystems_examples.subdir = examples
 module_qtsystems_examples.target = module-qtsystems-examples
 module_qtsystems_examples.depends = module_qtsystems_src
+module_qtsystems_examples.CONFIG = no_default_install
 !contains(QT_BUILD_PARTS,examples):!with-examples {
     module_qtsystems_examples.CONFIG = no_default_target no_default_install
     warning("No examples being used")
