diff --git a/src/imports/systeminfo/qsysteminfoimport.pri b/src/imports/systeminfo/qsysteminfoimport.pri
index ed64af6..396fa60 100644
--- a/src/imports/systeminfo/qsysteminfoimport.pri
+++ b/src/imports/systeminfo/qsysteminfoimport.pri
@@ -3,6 +3,8 @@ load(qt_module)
 TEMPLATE = lib
 CONFIG += qt plugin
 
+include(../../../modules/qt_systeminfo.pri)
+
 win32|mac:!wince*:!win32-msvc:!macx-xcode:CONFIG += debug_and_release
 
 isEmpty(TARGETPATH) {
